docs/en/deploy/Running-Alluxio-On-Docker.md

```diff
@@ -106,18 +106,17 @@ contents with
 $ docker exec ${container_id} cat /opt/alluxio/conf/alluxio-site.properties
 ```
 
-### Run in High-Availability Mode
+### Run in High-Availability Mode with Zookeeper
```

Comment:
This section specifically says with Zookeeper, but also mentioned the embedded journal. Is that intended?